tests/test_file_watcher.py

```diff
@@ -0,0 +1,48 @@
+import os
+import shutil
+import tempfile
+import time
+import unittest
+from collections import defaultdict
+
+from watchdog.observers import Observer
+
+from liste_files import FileFinallyCreatedEventHandler
+
+
+class TestExtractSubs(unittest.TestCase):
+    def test_watch_new_file(self):
+        tmp_dir = tempfile.mkdtemp()
+
+        created_file_count = defaultdict(lambda: 0)
+
+        def _increment_counter(name: str, dir: str):
+            created_file_count[os.path.join(dir, name)] += 1
+
+        observer = Observer()
+        file_watcher = FileFinallyCreatedEventHandler(['*.txt'], lambda name, dir: 'exclude' not in name,
```

Comment:
The var name contains `watcher` and the class name contains `Handler`. Looks like a mismatch to me.

Reply:
Yeah, rename :heavy_check_mark: